.circleci/config.yml

```diff
@@ -1,42 +1,94 @@
-version: 2.1
 jobs:
-  test:
+  build:
```

Comment:
Should this be "build and test" to be consistent with the workflow below?

Reply:
@yunjunz 
test works on my CircleCI, should also work here

I have another test project to trigger circleci with pull request. It has the same config and workflow. 

Probably something in Github setting should be changed.

Reply:
This is fine and is consistent with the workflow below. My mislook.